src/main/kotlin/gg/essential/elementa/markdown/drawables/HeaderDrawable.kt

```diff
@@ -12,6 +11,8 @@ class HeaderDrawable(
     private val paragraph: ParagraphDrawable
 ) : Drawable(md) {
     override val children: List<Drawable> get() = listOf(paragraph)
+    //TODO: Add custom ids 5/13/2022
+    internal val id = paragraph.textDrawables.joinToString { it.plainText() }.replace(' ', '-')
```

Comment:
I don't think we should be replacing spaces with dashes when we don't need to (that's just gonna make the live of users harder), especially given we don't implement all the other header-text-to-anchor-id transformations either.
Also not sure if the default joinToString separator (`, `) is appropriate, I would assume a simple space is closer to what the user would expect.